### PR TITLE
Added three enemy AI functions

### PR DESCRIPTION
**Pull request by Phoenix-neat**

### CODE REVIEW

File: Persona3FES/Modules/Common/Functions.json [hunk 1]
```diff
@@ -981,8 +981,8 @@
     {
         "Index": "0x0042",
         "ReturnType": "int",
-        "Name": "FUNCTION_0042",
-        "Description": "002C8050",
+        "Name": "PARTY_CHECK",
```
Comment (Tupelov) [inline Persona3FES/Modules/Common/Functions.json]: Change this to something like AI_CHK_PARTY

Comment (Tupelov) [reply Persona3FES/Modules/Common/Functions.json]: This might be AI_CHK_PLAYER_ID

File: Persona3FES/Modules/Common/Functions.json [hunk 2]
```diff
@@ -5427,8 +5427,8 @@
     {
         "Index": "0x0177",
         "ReturnType": "void",
-        "Name": "FUNCTION_0177",
-        "Description": "002D0830",
+        "Name": "AI_ACT_SKILL_WITH_PERSONA",
```
Comment (Tupelov) [inline Persona3FES/Modules/Common/Functions.json]: Maybe AI_ACT_PERSONA would be a better name

**Review by Tupelov — CHANGES_REQUESTED** [review]
Found some issues with naming consistency.

**Comment by Phoenix-neat** [comment]
let's do it again

File: Persona3FES/Modules/Common/Functions.json [hunk 3]
```diff
@@ -981,7 +981,7 @@
     {
         "Index": "0x0042",
         "ReturnType": "int",
-        "Name": "PARTY_CHECK",
+        "Name": "AI_CHK_PLAYER_ID",
```
Comment (Tupelov) [inline Persona3FES/Modules/Common/Functions.json:984]: Perhaps we could investigate which ID corresponds to which party member so we could create an enum.

File: Persona3FES/Modules/Common/Functions.json [hunk 4]
```diff
@@ -5427,7 +5427,7 @@
     {
         "Index": "0x0177",
         "ReturnType": "void",
-        "Name": "AI_ACT_SKILL_WITH_PERSONA",
+        "Name": "AI_ACT_PERSONA",
         "Description": "First value is Persona, second value is Skill",
```
Comment (Tupelov) [inline Persona3FES/Modules/Common/Functions.json]: This should go under the parameter descriptions, not the function itself

File: Persona3FES/Modules/Common/Functions.json (hunk 3 above)
Comment (Tupelov) [reply Persona3FES/Modules/Common/Functions.json:984]: Not important though

**Review by Tupelov — CHANGES_REQUESTED** [review]

File: Persona3FES/Modules/Common/Functions.json [hunk 5]
```diff
@@ -981,7 +981,7 @@
     {
         "Index": "0x0042",
         "ReturnType": "int",
-        "Name": "PARTY_CHECK",
+        "Name": "AI_CHK_PLAYER_ID",
         "Description": "Checks if the party member specified is present in the fight",
         "Parameters":
```
Comment (Tupelov) [inline Persona3FES/Modules/Common/Functions.json:986]: Add param names

File: Persona3FES/Modules/Common/Functions.json [hunk 6]
```diff
@@ -5427,7 +5427,7 @@
     {
         "Index": "0x0177",
         "ReturnType": "void",
-        "Name": "AI_ACT_SKILL_WITH_PERSONA",
+        "Name": "AI_ACT_PERSONA",
         "Description": "First value is Persona, second value is Skill",
         "Parameters":
```
Comment (Tupelov) [inline Persona3FES/Modules/Common/Functions.json:5432]: Add param names

**Review by Tupelov — CHANGES_REQUESTED** [review]
Needs param names as well.

File: Persona3FES/Modules/Common/Functions.json [hunk 7]
```diff
@@ -4573,7 +4573,7 @@
     {
         "Index": "0x013f",
         "ReturnType": "void",
-        "Name": "FUNCTION_013F",
+        "Name": "AI_XXX",
```
Comment (Tupelov) [inline Persona3FES/Modules/Common/Functions.json]: Put AI_0013F

**Review by Tupelov — CHANGES_REQUESTED** [review]
Almost ready just needs one thing

**Comment by Phoenix-neat** [comment]
yeah

**Pull request merged**